src/Evolution/Initialization/InitialData.hpp

```diff
@@ -24,7 +24,7 @@ decltype(auto) initial_data(const SolutionOrData& solution_or_data,
 
 /// \cond
 template <typename SolutionOrData, typename Coordinates, typename... Tags,
-          Requires<is_analytic_data_v<SolutionOrData>> = nullptr>
+          Requires<not is_analytic_solution_v<SolutionOrData>> = nullptr>
```

Comment:
Why not `is_analytic_data_v or is_numerical_data_v`? That would be easier to read IMO and probably safer?

Reply:
done